### PR TITLE
[Issue #45] Ignore clients auth redirection URLs if the request conta…

### PR DESCRIPTION
**Pull request by ferjm**
…ins an empty array

### CODE REVIEW

File: test/test_clients_api.js [hunk 1]
```diff
@@ -143,6 +161,14 @@ describe('Clients API', () => {
   });
 
   describe('GET ' + endpointPrefix + '/clients', () => {
+    beforeEach(done => {
+      db().then(models => {
+        models.Clients.destroy({
+          where: {}
```
Comment (julienw) [inline test/test_clients_api.js]: wondering it you actually need a where option ?

**Comment by julienw** [comment]
My only concern is that we can have either an empty array or a NULL that mean the same; I feel like we should try to normalize this in a Client instance: if the array is empty, then expose it as null.
What do you think ?

Otherwise there is a nit in the tests.

File: test/test_clients_api.js (hunk 1 above)
Comment (ferjm) [inline test/test_clients_api.js]: Yes, this is sequelize's way to specify that you want to clear the table.

**Comment by ferjm** [comment]
Makes sense, I'll update the PR. Thanks

File: test/test_clients_api.js [hunk 2]
```diff
@@ -143,6 +161,14 @@ describe('Clients API', () => {
...
+        }).then(() => { done() });
+      });
```
Comment (julienw) [inline test/test_clients_api.js]: nit: I realize you can do something simpler like:
```javascript
beforeEach(function() {
  return db().then(({ Clients }) => Clients.destroy({ where: {} }));
});
```

You can return promises from mocha's test functions :)

**Pull request merged**